### DISCUSSION TITLE
Page jumps to top

### DISCUSSION THREAD

**Discussion by TThrane** [fancyapps/ui#552] [General]
Hi
I'm using the latest verison on this page, but when I close the image, the page jumps to the top?
Seems to be an old issue, but I can't figure out how to stop it from doing that - any help?

Here's the page: https://film.thr.nu/film/min-ven-krokodillen - it's the poster image.

Regards
Thomas

**Comment by fancyapps** [comment]
Hi,

Fancybox does not have "page jump" issue unless you somehow mess with the layout. Why do you have `style="overflow-y: auto;"` for the HTML element? Remove that and it would work fine.

btw, your page has issue with the (horizontal) scrolling.

**Comment by TThrane** [comment]
Thank you!
I'm not sure where it gets that style from, but will figure it out.

I don't see any issue with horizontal scrolling?

**Reply by fancyapps** [reply]
> I don't see any issue with horizontal scrolling?

Your page has horizontal scrollbar on any browser.
